SortDisplay.py

```diff
@@ -176,5 +177,6 @@ def update():
         print disp.citems.swaps, "swaps"
         print disp.ccmp.comparisons, "comparisons"
 
+
```

Comment:
Please remove this extra line. :)

Reply:
You might want to have a look at PEP 8  ;)